etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table_national_accounts.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+from owid.catalog import Dataset, Table, TableMeta
+from owid.catalog.utils import underscore_table
+from owid.walden import Catalog as WaldenCatalog
+from structlog import get_logger
+
+from etl.helpers import Names
+from etl.paths import DATA_DIR, REFERENCE_DATASET
+from etl.steps.data.converters import convert_walden_metadata
+
+log = get_logger()
+
+# naming conventions
+N = Names(__file__)
+
+
+
+def run(dest_dir: str) -> None:
+    log.info("penn_world_table_national_accounts.start")
+
+    # retrieve raw data from walden
+    walden_ds = WaldenCatalog().find_one(
+        namespace="ggdc", short_name="penn_world_table_national_accounts", version="2021-06-18"
+    )
+    local_file = walden_ds.ensure_downloaded()
+
+    df = pd.read_excel(local_file, sheet_name="Data")
+
+    # # clean and transform data
+    # df = clean_data(df)
+
+    # create new dataset and reuse walden metadata
+    ds = Dataset.create_empty(dest_dir)
+    ds.metadata = convert_walden_metadata(walden_ds)
+    ds.metadata.version = "2022-11-28"
+
+    # create table with metadata from dataframe
+    table_metadata = TableMeta(
+        short_name=walden_ds.short_name,
+        title=walden_ds.name,
+        description=walden_ds.description,
+    )
+    tb = Table(df, metadata=table_metadata)
+
+    # underscore all table columns
+    tb = underscore_table(tb)
+    
+    ds.metadata.update_from_yaml(N.metadata_path, if_source_exists="replace")
+    tb.update_metadata_from_yaml(N.metadata_path, "penn_world_table_national_accounts")
+    
+    # add table to a dataset
+    ds.add(tb)
+
+    # finally save the dataset
+    ds.save()
+
+    log.info("penn_world_table_national_accounts.end")
+
+
+# def clean_data(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
I commented out the default `clean_data` function. Could this be related to the issue @pabloarosado?

Reply:
Comments are totally fine, [CI complains](https://buildkite.com/our-world-in-data/etl-unit-tests/builds/1751#0184bf26-b799-43b2-87cb-77dcde8b7c16) about 
```
==> Linting
--
  | etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table_national_accounts.py:8:1: F401 'etl.paths.DATA_DIR' imported but unused
  | etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table_national_accounts.py:8:1: F401 'etl.paths.REFERENCE_DATASET' imported but unused
  | etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table.py:8:1: F401 'etl.paths.DATA_DIR' imported but unused
  | etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table.py:8:1: F401 'etl.paths.REFERENCE_DATASET' imported but unused
```

To fix these just remove those `DATA_DIR` and `REFERENCE_DATASET` imports that are not used from [here](https://github.com/owid/etl/pull/638/files#diff-54b58125f916f5de9a946ab152eb6bee76729219ffdbf7ab14ddafaa6f4ff6c9R8) and [here](https://github.com/owid/etl/pull/638/files#diff-85085368856673c0c808e394cef6c8c064ab3be34d425be6f9d19a3fc995d792R8). Then run `make lint` to make sure those errors are fixed.

(These imports are generated by `walkthrough` and are only needed if you check that you want population. I should fix it and automatically remove them if population is not used.)

(In VSCode unused imports are greyed out which is really helpful)

Reply:
Thanks @Marigold. There are no errors for meadow now (I started to run walkthrough garden). Will commit the meadow changes.

```
==> Linting
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:8:68: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:12:66: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:13:97: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:18:23: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:20:64: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:30:57: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:31:57: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:42:86: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:46:74: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:47:86: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:48:90: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:53:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:58:3: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:58:3: F401 'google.colab' imported but unused
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:59:3: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:60:1: E722 do not use bare 'except'
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:61:3: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:65:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:70:1: E303 too many blank lines (3)
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:70:74: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:76:9: E117 over-indented
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:77:11: E114 indentation is not a multiple of 4 (comment)
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:78:11: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:78:52: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:80:11: E114 indentation is not a multiple of 4 (comment)
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:81:11: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:83:11: E114 indentation is not a multiple of 4 (comment)
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:84:11: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:86:11: E114 indentation is not a multiple of 4 (comment)
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:86:50: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:87:11: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:89:11: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:90:25: E128 continuation line under-indented for visual indent
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:91:25: E128 continuation line under-indented for visual indent
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:92:25: E128 continuation line under-indented for visual indent
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:93:1: W293 blank line contains whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:94:11: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:96:9: E722 do not use bare 'except'
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:97:11: E111 indentation is not a multiple of 4
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:103:1: E402 module level import not at top of file
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:104:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:106:1: E402 module level import not at top of file
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:131:90: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:134:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:136:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:137:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:138:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:139:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:140:28: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:142:24: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:143:22: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:144:20: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:145:28: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:146:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:147:24: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:149:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:150:22: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:151:22: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:158:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:159:22: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:160:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:161:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:162:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:163:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:164:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:165:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:166:26: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:174:29: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:175:29: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:176:29: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:177:29: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:178:31: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:185:86: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:188:98: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:192:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:193:33: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:193:44: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:200:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:205:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:206:56: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:210:36: E226 missing whitespace around arithmetic operator
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:210:49: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:215:22: E128 continuation line under-indented for visual indent
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:216:22: E128 continuation line under-indented for visual indent
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:217:22: E128 continuation line under-indented for visual indent
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:218:22: E128 continuation line under-indented for visual indent
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:221:34: E126 continuation line over-indented for hanging indent
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:221:85: E251 unexpected spaces around keyword / parameter equals
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:221:87: E251 unexpected spaces around keyword / parameter equals
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:224:39: E251 unexpected spaces around keyword / parameter equals
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:224:41: E251 unexpected spaces around keyword / parameter equals
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:224:44: E231 missing whitespace after ':'
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:224:71: E251 unexpected spaces around keyword / parameter equals
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:224:73: E251 unexpected spaces around keyword / parameter equals
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:227:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:228:95: E251 unexpected spaces around keyword / parameter equals
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:228:97: E251 unexpected spaces around keyword / parameter equals
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:230:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:233:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:243:70: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:259:1: W293 blank line contains whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:267:34: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:281:9: E116 unexpected indentation (comment)
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:281:9: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:283:1: W293 blank line contains whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:285:5: E303 too many blank lines (2)
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:287:5: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:293:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:294:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:298:13: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:311:1: W293 blank line contains whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:319:30: W291 trailing whitespace
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:329:5: E116 unexpected indentation (comment)
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:329:5: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/prepare_variables.py:335:1: E265 block comment should start with '# '
etl/steps/data/garden/ggdc/2022-11-28/penn_world_table.py:44:1: W293 blank line contains whitespace
etl/steps/data/garden/ggdc/2022-11-28/penn_world_table.py:47:1: W293 blank line contains whitespace
etl/steps/data/garden/ggdc/2022-11-28/penn_world_table_national_accounts.py:44:1: W293 blank line contains whitespace
etl/steps/data/garden/ggdc/2022-11-28/penn_world_table_national_accounts.py:47:1: W293 blank line contains whitespace
make: *** [default.mk:27: lint-default] Error 1
```